tasks/eth/base-001-MCP-L1/.env

```diff
@@ -0,0 +1,3 @@
+ETH_RPC_URL="https://gateway.tenderly.co/public/mainnet"
+OWNER_SAFE=0x7bB41C3008B3f03FE483B28b8DB90e19Cf07595c
+SAFE_NONCE=""
```

Comment:
We need to hardcode the foundation safe nonce here. Currently the expected ordering is:
- https://github.com/ethereum-optimism/superchain-ops/tree/main/tasks/eth/011-1-proto-ver%2Bownership-transfers will be executed first. The Foundation Operations Safe is used, at its current nonce of 92
- This playbook must be executed afterwards. The Foundation Operations Safe is against used, but with nonce 93. This requires specifying the nonce in the `.env` file here using the naming pattern shown in https://github.com/ethereum-optimism/superchain-ops/blob/10b7011977ba239b04f18bd8499e08c616c328a5/tasks/eth/013-fp-upgrade-fjord/.env#L7-L12

Reply:
(If the base Safe on that 2/2, `0x9855054731540A48b28990B63DcF4f33d8AE46A1`, plans to execute no transactions until this one, we can only hardcode the foundation nonce. But if Base will use that Safe to execute any transactions before this one, we need to hardcode its nonce too)

Reply:
Note that `council` and `foundation` are hardcodeds keywords in our `nested.just` file, so to avoid introducing changes, we should just treat base's safe as "council" and foundation's as "foundation". Check out https://github.com/ethereum-optimism/superchain-ops/blob/10b7011977ba239b04f18bd8499e08c616c328a5/tasks/eth/013-fp-upgrade-fjord/.env for an example of how to configure the `.env` file for a nested safe, and I'd suggest adding a comment explaining what's here and that "council" is really base's safe